roles/openshift_health_checker/openshift_checks/disk_availability.py

```diff
@@ -0,0 +1,121 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class DiskAvailability(OpenShiftCheck):
+    """Check that recommended disk space is available."""
+
+    name = "disk_availability"
+    tags = ["preflight"]
+
+    recommended_diskspace = {
+        "nodes": 15*10**9,
+        "masters": 40*10**9,
+        "etcd": 20*10**9,
+    }
+
+    minimum_docker_space_percent = 5.0
+    supported_docker_dev_driver = "devicemapper"
```

Comment:
Actually, remembering a very similar conversation I had with @rhcarvalho about this, it would be better to have the storage driver check be its own preflight check, allowing this PR and https://github.com/openshift/openshift-ansible/pull/3787 to focus on a single thing. WDYT?